students/Ruohan/lesson03/mailroom.py

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env python3
+
+''' simple mailroom proram for a non profit'''
+
+import sys
+
+datas = [['William Gates III',  653784.39, 2],
```

Comment:
you're storing the total and the number of donations -- while that's all we are asking for in the report, you may well want to keep the donation history -- better to keep a list of all donations, and compute the total when you need it.